pyproject.toml

```diff
@@ -15,10 +15,10 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-attrs = ">=18.2"
 aiobotocore= ">=0.10.0"
 boto3 = ">=1.9.49"
-aiohttp = ">=3.5.4"
+httpx = {version = "^0.11.1", extras = ["httpx"]}
```

Comment:
Does aws support parallel requests over HTTP/2? 😮

Reply:
httpx is used for h1 in this case. I just saw no reason to force aiohttp usage so you can bring your own http client and i added  built-in support for httpx and aiohttp.

Reply:
if it's bring-your-own, then, perhaps, these should be dev-deps, rather than deps? 🤔

Reply:
they're "extra" deps. you either install `aiodynamo[httpx]` or `aiodynamo[aiohttp]` or bring your own and install `aiodynamo`

Reply:
Ohhh that's how it works! :orz: